### ISSUE TITLE
[Feature]: Share prompt option | Random Prompt | Search for prompt

### ISSUE THREAD

**Issue by daisseur** [VoltaML/voltaML-fast-stable-diffusion#176]
The problem is my english is not very good then my prompt are awful.

## Share Prompt Option
I would like to have a "Share Prompt Option" (a checkbox maybe ?) to send the prompt on a db, maybe an existing or juste writing the prompt in  website each time you generate an image with the option activated.

## Random prompt chooser
A button or something else to pick a random prompt in the db maybe it could work with keyword ?

### Browse only prompt
There could be a section of the window to search for prompts related to the written one, and only text prompts.

### Browse prompt and image
Of course, the ideal would be to be able to propose an image to choose a prompt, but hosting so many images would be too expensive, so why not make it so that each computer/server running VoltaML can be referenced by a db/website to load images by making a GET request (the server would take care of not making accidental DDOS). 
All this would remain an option.

> DB/WEBSITE/ LOCAL FILETEXT / DB ?
If it's too difficult to create a database/website or to connect DeltaML to it, we could do it locally to at least facilitate indexing and browsing of this data

(Sorry For My english)

### Validations

- [X] Read the [docs](https://voltaml.github.io/voltaML-fast-stable-diffusion/).
- [X] Check that there isn't already an issue that asks for the same feature to avoid creating a duplicate.

**Comment by gabe56f** [comment]
With the share prompt option, I presume you're talking about sites like [mage.space](https://mage.space/), [lexica](https://lexica.art/) or [just plain CivitAI](https://civitai.com/) -- this could probably be rather easily done, I'll look into it later, currently focusing on animations

Random prompt chooser is something that can be looked into, however I'd much rather do something more akin to "dynamic" prompting, where someone could just type in, say, `{color} flower` and it the `{color}` part would be replaced with a custom color from a file somewhere in the `data/` folder. Meanwhile, what you could probably look at is enabling prompt-to-prompt in the small gearbox at the top right corner of the prompting part. That'll expand whatever you give it into a bigger "more rich" prompt and could generate interesting things.

The browse thing would be
a) a security risk
b) an integration hellhole

if we were, say, a big company, I'd say A is something we can deal with, however, even big companies would shy away from implementing something like this, solely because there's no way to actually make one that functions in every edge-case, hence, it being called "integration hellhole."

TL;DR: I'll look into the share button and the random prompt chooser, however the third suggestion is out-of-scope.

**Comment by Stax124** [comment]
I would stay out of anything that would require a connection to some server that we host, I just don't want to maintain another thing that in this case, would only cost me money. The only option would be integration with external sites like Civit, but as far as I know, they no longer maintain their API spec (or they moved it somewhere - might just be a skill issue on my side)
